src/main/java/core/db/DBUpdater.java

```diff
@@ -266,7 +266,8 @@ private void updateDBv500(int dbVersion) throws SQLException {
 			futureTrainingTable.tryDeleteColumn("WEEK");
 		}
 
-		if (columnExistsInTable("SourceSystem", MatchLineupTable.TABLENAME)) {
+		// SourceSystem does NOT exist in HO4, but HeimName does
+		if (columnExistsInTable("HeimName", MatchLineupTable.TABLENAME)) {
```

Comment:
@wsbrenk it means it will try the upgrade everytime we launch HO! isnt it?  (looking the PR from my iphone ...) 
We might need something like if matchtype does not exist ......... to make sure this is applied only once, no ?

Reply:
no  - since the column is deleted some lines lower

Reply:
i think it not a good idea to add the columns SourceSystem and then remove them again in the same method, if a user is starting upgrade from HO4.